projects/cashmere/src/lib/pipes/pipes.module.ts

```diff
@@ -1,9 +1,10 @@
 import {NgModule} from '@angular/core';
 
 import {EllipsisPipe} from './ellipsis.pipe';
+import { NullOrEmptyPipe } from './null-or-empty/null-or-empty.pipe';
 
 @NgModule({
-    declarations: [EllipsisPipe],
-    exports: [EllipsisPipe]
+    declarations: [EllipsisPipe, NullOrEmptyPipe],
```

Comment:
The more I think about it, the more I am starting to believe that each pipe should be able to be imported independently.  Perhaps we could have a `NullOrEmptyPipeModule` and an `EllipsisPipeModule` then have a `PipesModule` that `exports` all of them for convenience?